.github/workflows/pinot_vuln_check.yml

```diff
@@ -19,18 +19,8 @@
 
 name: Pinot Dependencies
 on:
-  push:
-    branches:
-      - master
-  pull_request:
-    branches:
-      - master
-    paths:
-      - "**/pom.xml"
-      - "**/package.json"
-      - "**/package-lock.json"
-      - "docker/images/pinot/**"
-      - ".github/workflows/**"
+  schedule:
```

Comment:
Why do we need a cron schedule? Let's still do triggered based?

Reply:
@walterddr trigger based on what ? can you tell me how?

Reply:
We were talking about changing this check to do not run on each push but run periodically in order to do not mark as erroneous PRs that do not actually introduce the vulnerability. On the other hand, we would lose the ability to detect when a PR adds a known vulnerability. Instead we would need to be notified by this periodic cron. 

The big question here is how are we going to know when this cron fails.